src/main/java/org/fanout/pubcontrol/PubControlClient.java

```diff
@@ -176,6 +186,9 @@ else if (this.authJwtClaim != null) {
             String token = Jwts.builder().setClaims(claims).
                     signWith(SignatureAlgorithm.HS256, decodedKey).compact();
             return "Bearer " + token;
+        } else if (this.authBearerKey != null) {
+            String keyString = new String(this.authBearerKey, StandardCharsets.UTF_8);
```

Comment:
as with the go lib, if we're always treating this as a string then consider making `authBearerKey` a string